src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -126,26 +132,39 @@ const TravelAidSection = ({
     fromQuayId,
   );
 
-  const situationsForFocused = focusedEstimatedCall.situations ?? [];
-  const noticesForFocused = getNoticesForServiceJourney(
-    serviceJourney,
-    focusedEstimatedCall.quay.id,
-  );
+  const situationsForFocused =
+    focusedEstimatedCall.situations.sort((n1, n2) =>
+      n1.id.localeCompare(n2.id),
+    ) ?? [];
+  const noticesForFocused =
+    getNoticesForServiceJourney(
+      serviceJourney,
+      focusedEstimatedCall.quay.id,
+    ).sort((n1, n2) => n1.id.localeCompare(n2.id)) ?? [];
+
+  const announcedSituationIds = situationsForFocused
+    .map((s) => s.id)
+    .filter(onlyUniques);
+  const announcedNoticeIds = noticesForFocused
+    .map((s) => s.id)
+    .filter(onlyUniques);
```

Comment:
Looks like this can be moved into `useTravelAidAnnouncements`. That way you don't have to pass them as props.